tests/components/yessssms/test_notify.py

```diff
@@ -1,7 +1,144 @@
 """The tests for the notify yessssms platform."""
+# pylint: disable=redefined-outer-name
 import unittest
+from unittest.mock import MagicMock
 import requests_mock
+import pytest
+
+from homeassistant.setup import async_setup_component
 import homeassistant.components.yessssms.notify as yessssms
+from homeassistant.components.yessssms.const import CONF_PROVIDER
+
+from homeassistant.const import CONF_PASSWORD, CONF_RECIPIENT, CONF_USERNAME
+
+
+@pytest.fixture
```

Comment:
If we name the fixtures with a different name than the fixture function name, we don't need to disable redefined-outer-name.

```py
@pytest.fixture(name="config")
```

Reply:
I was wondering about pylint on this.